build.proj

```diff
@@ -65,6 +66,7 @@
     <FunctionalTests Include="**/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj" />
     <FunctionalTestsProj Include="**/FunctionalTests/Microsoft.Data.SqlClient.Tests.csproj" />
 
+    <ManualTests Include="**/Common/Common.csproj" />
```

Comment:
🤔 What was the rationale for creating the new project rather than using the TestUtilities project?

I'll accept answers like "yeah I don't want to touch anything in the tools folder with a 10 foot pole, so I created a new one where we should put stuff going forward"